exporter/splunkhecexporter/logdata_to_splunk.go

```diff
@@ -62,12 +62,12 @@ func mapLogRecordToSplunkEvent(res pdata.Resource, lr pdata.LogRecord, config *C
 		case conventions.AttributeHostName:
 			host = v.StringVal()
 			fields[k] = v.StringVal()
-		case splunk.SourceLabel:
+		case splunk.DefaultSourceLabel:
```

Comment:
shouldn't it be something like `config.GetHostKey()` ? for the rest as well? just like how you have done on receiver side?

Reply:
I wanted to do that right next in a different PR to avoid a big PR. I didn't touch how the exporter works yet.